plugins/modules/system/ssh_config.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2015, Björn Andersson
+# Copyright: (c) 2021, Ansible Project
+# Copyright: (c) 2021, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ssh_config
+short_description: Manage SSH config for user
+version_added: '2.0.0'
+description:
+    - Configures SSH hosts with special C(IdentityFile)s and hostnames.
+author:
+    - Björn Andersson (@gaqzi)
+    - Abhijeet Kasurde (@Akasurde)
+options:
+  state:
+    description:
+      - Whether a host entry should exist or not.
+    default: present
+    choices: [ 'present', 'absent' ]
+    type: str
+  user:
+    description:
+      - Which user account this configuration file belongs to.
+      - If none given and I(ssh_config_file) is not specified, C(/etc/ssh/ssh_config) is used.
+      - If a user is given, C(~/.ssh/config) is used.
+      - Mutually exclusive with I(ssh_config_file).
+    type: str
+  group:
+    description:
+      - Which group this configuration file belongs to.
+      - If none given, I(user) is used.
+    type: str
+  host:
+    description:
+      - The endpoint this configuration is valid for.
+      - Can be an actual address on the internet or an alias that will
+        connect to the value of I(hostname).
+    required: true
+    type: str
+  hostname:
+    description:
+      - The actual host to connect to when connecting to the host defined.
+    type: str
+  port:
+    description:
+      - The actual port to connect to when connecting to the host defined.
+    type: str
```

Comment:
why str and not int?

Reply:
I kept `str` to match the original module and backward compatibility.